Portfolio/index.html

```diff
@@ -262,153 +261,448 @@
+    
+* {
+    margin: 0px;
+    padding: 0px;
+    border: 0px;
+    
+    /* h1 {
+        margin-left: 10px;
+    } */
 
 }
-.project-text1{
-    line-height: 20px;
-    margin-top: 90px;
-    margin-right: 119px;
-    float: right;
+.navbar{
+    width: 393%;
+}
+.navbar ul{
+    padding: 1px 102px;
+}
+.navbar ul li a {
+    padding: 0px 56px;
+    font-size: 1.5rem;
 }
+.navbar h1{
 
-.project2{
-    margin-left: 31px;
-    float: right;
-    position:static;
+    font-size: 4rem;
 }
-.project2 img{
-    float: right;
 
-    margin-top: -44px;
-    margin-right: 120px;
+
+.navbar img{
+    /* padding: 0px 0px; */
+    margin-left: -33px;
+   margin-top: 10px;
+   height: 28px;
+   width: 43px;
 }
-.project-text2{
-  display: contents;
-    /* border: 2px solid red; */
-    width: 90vw;
-    line-height: 20px;
-    float: left;
-    margin-top: 90px;
-    margin-left: 500px;
-}
-.project-text2 h1, b{
-    margin: 90px;
-width: 100vw;
-padding: 2px 34px;
-    display: contents;
-}
-.project3{
-    float: left;
-    position:static;
-    display: flex;
+
+.phnbtn button{
+    padding: 22px 63px;
+    margin-right:-12px;
+    width: 232px;
+}
+.phnbtn strong{
+    font-size: 0.8rem;
+}
+
+ 
+
+.gettoknowme{
+   position: relative;
+   display: inline-block;
+   margin-right: 50%;
+    justify-content: center;
+    align-items: center;
+    height: 138vh;
+    width: 404%;
 }
-.project3 img{
-    float: left;
+
+.gettoknowme p{
+    width: 54%;
+    justify-content: center;
+    align-items: center;
+    margin-top: 97px;
+    font-size: 3rem;
+    margin-left: 429px;
+}
+.name{
+    font-size: 3rem;
+        margin-left: 430px;
     
 }
+.gettoknowme h1{
+    margin-left: 577px;
+    width: 585px;
+    font-size: 5rem;
+    align-self: center;
+}
 
+.project-section{
+    margin-right: -1284px;
+}
 
-.project-text3{
+.containerss{
+    /* flex-wrap: wrap; */
+    justify-content: center;
+    align-items: center;
+    /* margin-left: 100px; */
+}
 
-    line-height: 20px;
-    margin-top: 90px;
-    margin-right: 119px;
-    float: right;
-}   
+.myskills{
+    margin-left: 838px;
+    background-color: beige;
+    
+}
 
-.project4{
-    float: right;
 
-    margin-top: 30px;
-    margin-right: 20px;
+.myskills h1 {
+    font-size: 5rem;
+    justify-content: center;
+    align-items: center;
+    margin-left: -171px;
+}
+.image-container img {
+	width: 50px;
+    filter: invert(60%);
+	height: auto;
+}
+.image-container h2 {
+font-size: 0.5rem;
 }
-.project4 img{
-    float: right;
 
-    margin-top: 30px;
-    margin-right: 120px;
+.John{
+    max-width: 100%;
+}
+.abtme{
+    width: 403vw;
+    height: 800px;
+}
+
+.abtme img{
+    height: 79%;
+    width: 48%;
 
 }
 
-.project-text4{
-    display: contents;
-    border: 2px solid red;
-    width: 90vw;
-    line-height: 20px;
-    float: left;
-    margin-top: 90px;
-    margin-left: 500px;
+.contact{
+    padding: 20px 13px;
+     width: 400%;
 }

```

Comment:
The closing tag for '<div>' within the '.project' section should come after the corresponding '<button>' tag.